sepal_ui/scripts/decorator.py

```diff
@@ -62,8 +60,11 @@ def init_ee() -> None:
 ################################################################################
 
 
+@deprecated(
+    version="3.1", reason="debug argument defaults to true. Will be removed in v3.2"
+)
```

Comment:
ok sorry, I made a mistake here. using @deprecated decorator will actually raise the deprecation warning everytime you read the method and not only when you set the debug parameter manually. 

I think instead you should do something like: 
```python 
def catch_error(alert: Optional[v.Alert] = None, debug: Optional[bool] = None): 
    # check if the deprecated parameter is set
    if debug is not None: 
        warn("debug argument defaults to ``true``. Will be removed in v3.2")
    debug = True
    # do stuff as usal
```

same goes for `loading_button`